pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { useRouter } from 'next/router';
 import { Box } from '@material-ui/core';
 import { ContactFilters } from '../../../src/components/Contacts/ContactFilters/ContactFilters';
 import { ContactsTable } from '../../../src/components/Contacts/ContactsTable/ContactsTable';
+import { ContactDetails } from '../../../src/components/Contacts/ContactDetails/ContactDetails';
 
 const ContactsPage: React.FC = () => {
   const { t } = useTranslation();
   const {
     query: { accountListId },
   } = useRouter();
 
+  const [contactDetailsId, setContactDetailsId] = useState<string | null>(null);
+  const [contactDetailsHidden, setContactDetailsHidden] = useState<boolean>(
+    true,
+  );
```

Comment:
Do we need to keep `contactDetailsId` around when it's hidden? Or could we just clear `contactDetailsId` instead and use a single state value for both of these usages?

The current version takes 2 clicks to switch from one contact to the next since it's toggling the hidden state every time.

Reply:
Yeah that could be nice.  Hiding the panel could be controlled by whether or not `contactDetailsId` is defined or not.